xx.md

```diff
@@ -0,0 +1,56 @@
+NIP-XX
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines a new event kind `31337` representing an audio track. These events are _parameterized replaceable_
+and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+A kind `31337` event's `content` should be summary or description of the audio content.
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in [NIP 92](./92.md). Multiple `imeta` tags MAY be included to provide for different
+  use cases (e.g. streaming vs download).
+- `title` is the title of the audio track
+- `subject` is the title of the audio track (deprecated, but required for compatibility)
+
+The following tags are optional:
+
+- `i` is an external GUID.
```

Comment:
What is the distinction between this tag and the `d` tag? In other words, what are some examples of needing both?

Reply:
I originally had it in the `d` tag, but thought it might be useful to be able to change the GUID if a different standard is chosen, or include multiple guids (e.g. music vs podcast if a track qualifies as both).

Reply:
I recall there being an actual `["d", <value>]` (in the deprecated NIP-33), but I don't even see a reference to a `d` tag in NIP-01. 

Are `d` tags used at all? Looks like the `d` tag value only exists within the `a` tag, but not as an actual tag itself.

It is only mentioned here in NIP-01:

>The a tag, used to refer to a (maybe parameterized) replaceable event
for a parameterized replaceable event: `["a", <kind integer>:<32-bytes lowercase hex of a pubkey>:<d tag value>, <recommended relay URL, optional>]`

>for kind n such that 30000 <= n < 40000, events are parameterized replaceable, which means that, for each combination of pubkey, kind and the d tag's first value, only the latest event MUST be stored by relays, older versions MAY be discarded.

Reply:
so is an `i` tag basically the new `d` tag? Or does NIP-01 need to be updated to explicitly define the `d` tag?

Reply:
`i` and `d` might be the same in some cases, but emerging best practices are to make `d` tags random UUIDs with no semantic meaning. `i` is a reference to an external source. In this case, it might even be beneficial to have more than one (multiple `d` tags would result in undefined behavior).

Reply:
So if I understand you correctly, it sounds like the `d` tag is mainly used as a way to identify the content contained in the event, and the `i` tag(s) can be used to reference related content?

The way I see us using this is to put our internal content identifier guids in the `d` tag and then use the `i` tag(s) for any related content. For example, if a podcast contained some songs, the guid for the podcast would go in the `d` tag and the song guids in the `i` tags. At this time I don't see any reason to create and use another random identifier for `d`

Reply:
Now I wonder if `i` makes sense here. Based on NIP-39 and the `i` tag being used for identities, I don't think it makes sense to use this tag for arbitrary id's. I'm still not totally clear on the use case for `i`

As for referencing other tracks that this piece of content might contain, that sounds like something we could add later as an extension of this NIP. I can see the eventual benefit but trying to get there now might add unnecessary complexity to this proposal.

Reply:
This is a different context, so it's ok if the meaning of `i` varies a bit. The sematics of tags are defined by the kind the tag is applied to. We only have so many one-letter tags, better to re-use i than introduce a new one.